infra/local/opensearch/Dockerfile

```
@@ -0,0 +1,6 @@
+# Local OpenSearch with the analysis-icu plugin baked in. The channels index
+# mapping uses a custom `content_text` analyzer whose `icu_folding` filter comes
+# from analysis-icu, which the stock image does not ship. Baking it at build time
+# (vs installing on every container start) keeps startup fast and offline-safe.
+FROM opensearchproject/opensearch:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)